### ISSUE TITLE
Crashed while working on a board. Can't recover the campaign.

### ISSUE THREAD

**Issue by pyrestriker** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#674]
**Describe the bug**
I was placing some assets, fumbled my thumb into some keys and it crashed. Now the whole campaign won't load. I was working on a different board than the default one that loads when opening the campaign.

**To Reproduce**
No idea. I was working on a board and it crashed and now won't recover dispite the default board is a different board than the one I was working on. What I do know is it crashed the first time when I accidentally fumbled my thumb into Win+Ctrl+Shift all at once.

**Screenshots/Videos/Twitch Clips**

Video of crash
https://youtu.be/xEoyeaacvvM

Player.log
[Player.log](https://github.com/Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker/files/6247554/Player.log)
Previous player log
[Player-prev.log](https://github.com/Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker/files/6247555/Player-prev.log)

And crash files from today.
[Crash_2021-04-02_025819550.zip](https://github.com/Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker/files/6247550/Crash_2021-04-02_025819550.zip)
[Crash_2021-04-02_030017627.zip](https://github.com/Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker/files/6247551/Crash_2021-04-02_030017627.zip)
[Crash_2021-04-02_030325806.zip](https://github.com/Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker/files/6247552/Crash_2021-04-02_030325806.zip)
[First_Crash_2021-04-02_025729217.zip](https://github.com/Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker/files/6247553/First_Crash_2021-04-02_025729217.zip)

**Additional context**
Really hoping I didn't just lose my whole campaign.

**Comment by pyrestriker** [comment]
I forced it to load! I was able to get it to load by switching real quick to a new board during loading. Unlike Dragon Build, Chimera Build lets you open the fly-out menus while the board is still loading. However, swapping to the wagon room immediately crashes it. Swapping back to the default board (Crystal Valley) worked, and this is the only thing I found out of place. Might be worth noting this is the same door I used in the Wagon Room board?
![image](https://user-images.githubusercontent.com/3858960/113383956-d91e6980-9339-11eb-967d-172e850e616a.png)
Latest crash files in case any new data. Same lib_burst_generated.dll error from what I see.
[Crash_2021-04-02_052222552.zip](https://github.com/Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker/files/6247841/Crash_2021-04-02_052222552.zip)

**Comment by pyrestriker** [comment]
So I posted the last comment prematurely. More oddness but it looks like I fixed it eventually. Tried clicking on the misplaced door, nothing happened. Tried moving the lantern out of the way to make room and check for hitboxes and it went away, but then a single door was spinning on the wrong axis on its own? I got it on video. Moving the entire cart seems to have fixed it for now.

Video of it happening: https://youtu.be/RI6dJzeOXfw
